data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -39,23 +39,20 @@ spec:
             - /bin/bash
             - -ce
             - |
-              echo 'Installing bridge and tuning CNIs'
-              cni_mount_dir=/opt/cni/bin
-              sourcebinpath=/usr/src/github.com/containernetworking/plugins/bin
-              cp --remove-destination ${sourcebinpath}/bridge ${cni_mount_dir}/cnv-bridge
-              cp --remove-destination ${sourcebinpath}/tuning ${cni_mount_dir}/cnv-tuning
+              if find ${cni_mount_dir}/bridge &>/dev/null; then
```

Comment:
This won't be enough if the comment below gets addressed.

Reply:
changed https://github.com/kubevirt/cluster-network-addons-operator/compare/cd82a3236f8ffba41a0862f91456e7b1c7518811..419fedfbc1b9b72932e6c4ec63b35d836c2100f1

but willing to hear more ideas, thanks

Reply:
addressed the comment below
can you please explain why it is not enough?
any other alternatives?

Reply:
With this condition we would never update the plugins.

On the first run of CNAO on a Kubernetes cluster, the else branch would install `bridge` and `cnv-bridge`. On any consecutive run, this condition would be evaluated as true. Thus, we will never update existing binaries with newer ones.

I simple solution for this would be to include a "flag" in the container image. Our upstream container image would stay as is. Any downstream image that wants to only symlink could `touch /opt/symlink-cnis`. The condition in this daemon set would look for that file.

Reply:
interesting,
but would be better to touch only upstream if possible,
we can add some kind of mechanism
that if at the first time there was no `bridge`, it touches `${cni_mount_dir}/upstream`
with this we can change the condition to be `&&` the above file doesn't exist
wdyt?

Reply:
Let's try it an see how it looks

Reply:
instead of a flag, we can check if the `bridge` is a real file or softlink, wdyt?
less files is better